mappings/net/minecraft/village/VillagerProfession.mapping

```diff
@@ -24,3 +24,4 @@ CLASS net/minecraft/class_3852 net/minecraft/village/VillagerProfession
 	METHOD method_19199 getGatherableItems ()Lcom/google/common/collect/ImmutableSet;
 	METHOD method_19630 getSecondaryJobSites ()Lcom/google/common/collect/ImmutableSet;
 	METHOD method_22384 getWorkSound ()Lnet/minecraft/class_3414;
+	METHOD method_35202 getId ()Ljava/lang/String;
```

Comment:
Is this the full id like `minecraft:nitwit`? The professions already have ids in the registry

Reply:
No namespaces on the vanilla ones, just returns `this.id`, the same as the `toString()` method of the class